tests/test_external_calculators.py

```
@@ -2,28 +2,38 @@
     try:
         calc = MLIPEnum[model.name].value()
-    except (LocalTokenNotFoundError, HTTPError, FileNotFoundError) as e:
+    except (
+        LocalTokenNotFoundError,
+        GatedRepoError,
+        HTTPError,
+        HTTPStatusError,
+        FileNotFoundError,
+    ) as e:
         pytest.skip(str(e))
+    except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)